react-tic-tac-toe/README.md

```diff
@@ -0,0 +1,14 @@
+# React Tic Tac Toe
+
+A game example built using [TypeScript](https://github.com/Microsoft/TypeScript) and [React](https://github.com/facebook/react), following guidelines from [react-webpack guide](https://github.com/Microsoft/TypeScript-Handbook/blob/master/pages/quick-start/react-webpack.md) from TypeScript handbook.
+
+## Build
+```
+npm install -g typescript webpack
```

Comment:
I included the .d.ts in the project since react's version is specified in package.json while typings has no version control. The sample may break from future non-backward-compatible .d.ts if I have the audience download the newest .d.ts through ```typings install```. Does that make sense?